pkgs/development/python-modules/pwntools/default.nix

```diff
@@ -17,6 +17,7 @@ buildPythonPackage rec {
   propagatedBuildInputs = [ Mako packaging pysocks pygments ROPGadget capstone paramiko pip psutil pyelftools pypandoc pyserial dateutil requests tox pandoc unicorn intervaltree ];
 
   disabled = isPy3k;
+  doCheck = false;
```

Comment:
Could you just add a comment here stating the archive doesn't include tests.

Reply:
Done.